dockerfiles/django/Dockerfile-dev

```diff
@@ -39,7 +40,18 @@ COPY . /code
 
 # RUN python -m manage tailwind --skip-checks download_cli
 
-RUN DATABASE_URL=sqlite://:memory: SECRET_KEY=build-key REDIS_URL= python manage.py tailwind --skip-checks download_cli
+# install standalone tawildind cli and cache it
+
+RUN --mount=type=cache,target=/tmp/tailwind-cache \
+    if [ ! -f /tmp/tailwind-cache/tailwindcss ]; then \
+        curl -L https://github.com/tailwindlabs/tailwindcss/releases/download/v4.0.0-beta.1/tailwindcss-linux-x64 -o /tmp/tailwind-cache/tailwindcss; \
+    fi && \
+    cp /tmp/tailwind-cache/tailwindcss /usr/local/bin/tailwind && \
+    chmod +x /usr/local/bin/tailwind
```

Comment:
To be honest, this shouldn't be needed anymore, given the project is updated to a recent version of django-tailwind-cli. Line 41 should take care of it.